SilMock/Google/Service/Directory/UsersResource.php

```diff
@@ -361,15 +361,15 @@ public function listUsers($parameters = [])
                 $familyName = $nameEntry['familyName'] ?? null;
                 $fullName = $nameEntry['fullName'] ?? trim($givenName . ' ' . $familyName);
                 $newName = new \Google_Service_Directory_UserName([
-                    'familyName' => $familyName,
-                    'fullName'   => $fullName,
-                    'givenName'  => $givenName,
-                ]);
+                                                                      'familyName' => $familyName,
+                                                                      'fullName'   => $fullName,
+                                                                      'givenName'  => $givenName,
+                                                                  ]);
```

Comment:
Personally, I liked it how it was before better. I don't touch this enough to care much though.

Reply:
when I select whole file and cut and paste it back... it reformats some things ugly like this.